firecrown/connector/cosmosis/likelihood.py

```diff
@@ -143,6 +143,8 @@ def execute(self, sample: cosmosis.datablock) -> int:
             # Cosmology will be generated by CCL. We want to be sure that CosmoSIS
             # has not also been configured to use CAMB.
             self.tools.prepare()
+
+        handle_unused_params(params=firecrown_params, raise_on_unused=False)
```

Comment:
I'd argue for defaulting to `raise_on_unused=True` and getting `raise_on_unused` from `config` to override by the user.